apps/api/src/app/subscribers/dtos/chat-oauth-request.dto.ts

```diff
@@ -7,6 +7,10 @@ export class ChatOauthRequestDto {
   @IsMongoId()
   @IsDefined()
   environmentId: string;
+
+  @IsOptional()
+  @IsString()
+  integrationIdentifier?: string;
```

Comment:
The chat OAuth flow - allow providing the `integration identifier` that will be used to find the exact integration, then the chat channel credentials are stored under the subscriber credentials array for that specific integration.

If not provided then we will be looking for the latest created active integration, which is backward compatible change.